samples/tiled-convolution/CMakeLists.txt

```diff
@@ -1,4 +1,28 @@
 set(SOURCE_NAME "tiled-convolution")
+
+if(DEFINED TARGET)
+message(STATUS "TARGET is defined")
+  # If the user has specified  RCAR as a target backend
+  #the local size will be set to 1 and 32
+  if(${TARGET} STREQUAL "RCAR")
+    message(STATUS "${TARGET} device is chosen")
+    set(LOCAL_SIZE_M 1)
+    set(LOCAL_SIZE_N 32)
+  endif()
+else()
+  if (NOT DEFINED LOCAL_SIZE_M)
+    set(LOCAL_SIZE_M 16)
+  endif()
+
+  if (NOT DEFINED LOCAL_SIZE_N)
+    set(LOCAL_SIZE_N 16)
+  endif()
+
+endif()
+
+add_definitions(-DLOCAL_SIZE_M=${LOCAL_SIZE_M})
```

Comment:
Can you replace both `add_definitions()` with a single `target_compile_definitions(${SOURCE_NAME} PRIVATE LOCAL_SIZE_M=${LOCAL_SIZE_M} LOCAL_SIZE_N=${LOCAL_SIZE_N})` please?